File: JavaWeb2-eureka-client-user/src/main/java/com/javaweb/web/eo/dictionary/DictionaryListRequest.java
package com.javaweb.web.eo.dictionary;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DictionaryListRequest implements Serializable {
    
	private static final long serialVersionUID = 1421848003510714297L;

	private String dataType;//数据类型
    
    private String keyCode;//key值
    
    private String means;//含义
    
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条
    
}

File: JavaWeb2-angular/src/app/model/dictionary/DictionaryListRequest.ts
import {CommonConstant} from '../../constant/CommonConstant';

export class DictionaryListRequest {

  dataType:string;//数据类型

  keyCode:string;//key值

  means:string;//含义

  createStartDate:any = null;//字典创建的开始日期

  createEndDate:any = null;//字典创建的结束日期

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}
